src/constants/geoChart.ts

```diff
@@ -73,3 +73,5 @@ export const PUSHPIN_STYLE_CIRCLE = "circle";
 export const PUSHPIN_STYLE_CIRCLE_COLOR = "circle-color";
 export const PUSHPIN_STYLE_CIRCLE_SIZE = "circle-radius";
 export const PUSHPIN_STYLE_CIRCLE_STROKE_COLOR = "circle-stroke-color";
+export const EMPTY_SEGMENT_VALUE = "empty-segment-filter";
+export const EMPTY_SEGMENT_ITEM = "empty_segment_item";
```

Comment:
why didn’t you use `empty-segment-item`?

Reply:
fixed